prow/github/client.go

```diff
@@ -1022,6 +1024,40 @@ func (c *client) CreateRepoHook(org, repo string, req HookRequest) (int, error)
 	return c.createHook(org, &repo, req)
 }
 
+func (c *client) deleteHook(org string, repo *string, id int, req HookRequest) error {
```

Comment:
nit: It would be a little simpler to pass the path to `deleteHook` instead of the org and repo since the callers are already org or repo specific.

Reply:
Done. It's indeed much simpler.